web/core/components/profile/overview/workload.tsx

```
@@ -8,34 +9,37 @@ type Props = {
-          </a>
-        </div>
-      ))}
+export const ProfileWorkload: React.FC<Props> = ({ stateDistribution }) => {
+  const { t } = useTranslation();
+  return (
+    <div className="space-y-2">
+      <h3 className="text-lg font-medium">{t("Workload")}</h3>
+      <div className="grid grid-cols-1 justify-stretch gap-4 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-5">
+        {stateDistribution.map((group) => (
+          <div key={group.state_group}>
+            <a>
```

Biome (2.1.2)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)